meilisearch-core/src/update/stop_words_deletion.rs

```diff
@@ -101,14 +101,18 @@ pub fn apply_stop_words_deletion(
 
     // now that we have setup the stop words
     // lets reindex everything...
-    reindex_all_documents(
-        writer,
-        main_store,
-        documents_fields_store,
-        documents_fields_counts_store,
-        postings_lists_store,
-        docs_words_store,
-    )?;
+    if let Ok(number) = main_store.number_of_documents(writer) {
```

Comment:
I do not understand what would be the impact of this change.

Reply:
no documents => nothing to reindex and also no problem of missing schema

Reply:
Ho I understand now, the first step of the `reindex_all_documents` is to gather the schema and removing stop-words triggers a reindexing. Thank you!